src/main/java/com/ait/lienzo/client/core/shape/wires/layout/direction/VerticalLayoutFactory.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ait.lienzo.client.core.shape.wires.layout.direction;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.ait.lienzo.client.core.shape.wires.layout.direction.DirectionLayout.Direction;
+import com.ait.lienzo.client.core.shape.wires.layout.direction.DirectionLayout.Orientation;
+import com.ait.lienzo.client.core.shape.wires.layout.direction.DirectionLayout.ReferencePosition;
+import com.ait.lienzo.client.core.shape.wires.layout.direction.DirectionLayout.VerticalAlignment;
+import com.ait.lienzo.client.core.types.BoundingBox;
+import com.ait.tooling.common.api.java.util.function.Function;
+
+public class VerticalLayoutFactory
+{
+
+    private static Map<ReferencePosition, DirectionLayoutBuilder<VerticalAlignment>> builders = new HashMap<ReferencePosition, DirectionLayoutBuilder<VerticalAlignment>>()
+    {{
```

Comment:
double-brace syntax, let's wait for @hasys to confirm if we should refactor this...

Reply:
:+1: